src/vm/fieldmarshaler.h

```diff
@@ -134,34 +144,27 @@ BOOL IsStructMarshalable(TypeHandle th);
 //
 // Each redirected field gets one entry in LayoutRawFieldInfo.
 // The array is terminated by one dummy record whose m_MD == mdMemberDefNil.
-// WARNING!! Before you change this struct see the comment above the m_FieldMarshaler field
 //=======================================================================
 struct LayoutRawFieldInfo
 {
     mdFieldDef  m_MD;             // mdMemberDefNil for end of array
     UINT8       m_nft;            // NFT_* value
-    UINT32      m_offset;         // native offset of field
-    UINT32      m_cbNativeSize;   // native size of field in bytes
+    RawFieldPlacementInfo m_nativePlacement; // Description of the native field placement
     ULONG       m_sequence;       // sequence # from metadata
-    BOOL        m_fIsOverlapped;
 
 
     //----- Post v1.0 addition: The LayoutKind.Sequential attribute now affects managed layout as well.
     //----- So we need to keep a parallel set of layout data for the managed side. The Size and AlignmentReq
     //----- is redundant since we can figure it out from the sig but since we're already accessing the sig
     //----- in ParseNativeType, we might as well capture it at that time.
-    UINT32      m_managedSize;    // managed size of field
-    UINT32      m_managedAlignmentReq; // natural alignment of field
-    UINT32      m_managedOffset;  // managed offset of field
-    UINT32      m_pad;            // needed to keep m_FieldMarshaler 8-byte aligned
+    RawFieldPlacementInfo m_managedPlacement;
 
     // WARNING!
     // We in-place create a field marshaler in the following
     // memory, so keep it 8-byte aligned or 
     // the vtable pointer initialization will cause a 
     // misaligned memory write on IA64.
```

Comment:
IA64 is no longer a thing. I would instead just state it should be 8 byte aligned.